packages/gatsby/rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import babel from 'rollup-plugin-babel';
+import autoExternal from 'rollup-plugin-auto-external';
+
+export default {
+  input: 'cache-dir/gatsby-browser-entry.js',
+  plugins: [
+    autoExternal(),
+    babel({
+      presets: [[require(`path`).join(__dirname, `..`, `..`, `.babel-preset.js`), {modules: false}]],
+      runtimeHelpers: true,
+      exclude: ['./node_modules/**']
+    })
+  ],
+  output: [
+    {file: 'lib/index.cjs.js', format: 'cjs', sourcemap: true},
```

Comment:
the files produced here are the ones that should be referenced as `main` and `module` in the `package.json`. 

the directory and file names can certainly be adjusted to fit your conventions, but i started with these to demonstrate the mechanics to start with.